CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Braintree iOS SDK Release Notes
 
 ## unreleased
+* BraintreeCore
```

Comment:
It seems like this branch has an outdated version of what's on `paypal-app-switch-feature` you may want to pull down the latest from the feature branch and then merge it in. These entries on the latest version will be replaces with the [diff from this PR in the CHANGELOG](https://github.com/braintree/braintree_ios/pull/1232/files).